src/screens/Ticketing/Ticket/Details/QrCode.tsx

```diff
@@ -123,39 +160,39 @@ const useQrCode = (
 
   return tokenQRCode;
 };
-const QrCodeError = ({retry}: {retry?: (forceRestart: boolean) => void}) => {
-  const {t} = useTranslation();
 
-  return (
-    <Sections.GenericItem>
-      <MessageBox
-        type={'error'}
-        title={t(TicketTexts.details.qrCodeErrors.generic.title)}
-        message={t(TicketTexts.details.qrCodeErrors.generic.text)}
-        onPress={retry && (() => retry(true))}
-        onPressText={retry && t(TicketTexts.details.qrCodeErrors.generic.retry)}
-      />
-    </Sections.GenericItem>
-  );
-};
-const QrCodeDeviceNotInspectable = ({
-  inspectableToken,
-}: {
-  inspectableToken: RemoteToken;
-}) => {
+// const QrCodeError = ({retry}: {retry?: (forceRestart: boolean) => void}) => {
+//   const {t} = useTranslation();
+//
+//   return (
+//     <Sections.GenericItem>
+//       <MessageBox
+//         type={'error'}
+//         title={t(TicketTexts.details.qrCodeErrors.generic.title)}
+//         message={t(TicketTexts.details.qrCodeErrors.generic.text)}
+//         onPress={retry && (() => retry(true))}
+//         onPressText={retry && t(TicketTexts.details.qrCodeErrors.generic.retry)}
+//       />
+//     </Sections.GenericItem>
+//   );
+// };
```

Comment:
Perhaps remove all these comments if are not going to be used.

Reply:
I actually will keep this for now. We have Figma design for the error, we have been through deciding norwegian/english texts for the error, and it is ment to be readded when fallback is removed. If this is still not reenabled in 2023 you can say "I told you so" 😅